com.unity.template-universal/Packages/com.unity.template.universal/package.json

```diff
@@ -7,6 +7,14 @@
   "host": "hub",
   "description": "The Universal Render Pipeline Template configures Project settings for Projects where performance, wide platform support, and ease of customizing graphics are the primary considerations.\n\nThis Template uses the Universal Render Pipeline (URP) and Shader Graph. URP is prebuilt Scriptable Render Pipeline that is quick and easy to customize, and lets you create optimized graphics across a wide range of platforms. URP also includes an optimized 2D renderer complete with 2D lights and pixel perfect rendering, and an integrated post-processing solution. Shader Graph is a tool that allows you to create shaders using a visual node editor instead of writing code.\n\nThis template contains a sample Scene that contains examples of how to configure lighting settings, Materials, Shaders, and post-processing effects in URP, several preconfigured Universal Render Pipeline Assets that let you quickly swap between graphics quality levels, and Presets that have been optimized for use with URP.",
   "dependencies": {
-    "com.unity.render-pipelines.universal": "13.0.0"
+    "com.unity.2d.sprite": "1.0.0",
+    "com.unity.2d.tilemap": "1.0.0",
```

Comment:
I see these packages have been in the `manifest.json` since before this PR, but the `package.json` didn't have them before and they are not injected by default - so they technically haven't been part of URP Sample before, is this an intentional change?

Reply:
@phi-lira  / @Verasl  any thought?

Reply:
Ah weird, I think they should be there, will make much more sense once we have the 2D renderer being used in the new Template, but for now I think this should indeed be added.

Reply:
I don't follow why we have some package dependencies duplicated in both `manifest.json` and `packages.json` is that expected? I'd expect that the dependencies in package.json should not be in manifest.json as the URP template package will import them indirectly anyway. If this is correct, the duplicated packages in manifest.json are redundant and should be removed.

Reply:
packages.json is used when we develop,  manifest.json is used when we bundle the template into a packages and is the real thing that matter for users. But the test currently check that both are in sync

Reply:
I believe it's the opposite re: package.json & manifest.json - manifest.json is essentially used for development and package.json is what the end users will see. This is why I wanted to double check that we wanted to add 2D packages, since they haven't been included in package.json before and therefore have not been a part of the template from the looks of it.

Template tests are checking that the packages are duplicated between manifest.json and package.json like Sebastien mentioned (with the exception of preview packages - these are only allowed in manifest.json), so it is expected for the packages to be duplicated between the two files.

Reply:
let those package for universal then, it don't harm and many people start from the template for new project (yes I know there is blank template now :))

Reply:
I'll approve as I see no risk here, but would like to follow up on package.json and manifest.json and why it needs to be duplicated.